src/scripts/main.js

```diff
@@ -1,3 +1,267 @@
 'use strict';
 
-// write your code here
+const theOnlyButton = document.querySelector('.button');
+const board = document.querySelector('tbody');
+const boardSize = 4;
+const pointsCounter = document.querySelector('.game-score');
+const messageForWinner = document.querySelector('.message-win');
+const messageForLooser = document.querySelector('.message-lose');
+const allCells = document.querySelectorAll('td');
+const winnerScore = 2048;
+let score = 0;
+
+theOnlyButton.addEventListener('click', () => {
+  changeButtonClass();
+  hideStartMessage();
+
+  [...allCells].forEach(cell => {
+    cell.innerHTML = '';
+    updateClassList(cell);
+
+    return cell;
+  });
+
+  if (messageForLooser.classList.contains('hidden') === false) {
```

Comment:
```suggestion
  if (!messageForLooser.classList.contains('hidden')) {
```
You should change such comparisons inside `if` everywhere.